straxen/plugins/led_cal/led_calibration.py

```diff
@@ -122,48 +178,130 @@ def get_records(raw_records, baseline_window):
     mask = np.where((records["record_i"] == 0) & (records["length"] == 160))[0]
```

Comment:
Although it was like that in the previous version, it will be better if we could add a URLConfig for it instead of using the hard coded number

Reply:
fixed

Reply:
seems that the thrid argument of `get_records` should be `led_cal_record_length`, but a same variable is defined inside the function itself. Please check my commit to see if it's the right fix: https://github.com/XENONnT/straxen/pull/1401/commits/c08d6c00827881c9afb7704c9863c1a22d8e378e

Reply:
@tflehmke any update here?

Reply:
Hey, it seems I forgot to commit push that change. I had done the same thing as you did, but also renamed the `record_length` for clarity.